src/vm/mda.cpp

```diff
@@ -1,4017 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I agree that we may want to consider bringing some of the MDAs back - it is why I have not checked `FEATURE_MDA` in https://github.com/dotnet/coreclr/issues/9067.